falco/assets/logs/falco.yaml

```diff
@@ -0,0 +1,61 @@
+id: falco
+metric_id: falco
+backend_only: false
+facets:
+  - groups:
+      - User
+    name: User Name
+    path: usr.name
+    source: log
+  - groups:
+      - Event
+    name: Event Name
+    path: evt.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Falco
+  enabled: true
+  filter:
+    query: source:falco
+  processors:
+    - type: date-remapper
+      name: Define `time` as the official date of the log
+      enabled: true
+      sources:
+        - time
+    - name: Lookup on `priority` to `status`
+      enabled: true
+      source: priority
+      target: status
+      lookupTable: |-
+        Warning, warning
+        Notice, warning
```

Comment:
While testing the changes, I just noticed this: shouldn't `Notice` by mapped to `notice` instead?

Reply:
We have mapped it with `notice`.